R/shiny_functions.R

```diff
@@ -11,28 +11,33 @@
 
 qi_table <- function(table_data, selected_units, config) {
   national <- table_data[["national"]]
-  if (!rlang::is_empty(table_data[["RHF"]])) {
-    colnames(table_data[["RHF"]])[
-      colnames(table_data[["RHF"]]) == "OrgNrRHF"] <- "OrgNr"
-    colnames(table_data[["RHF"]])[
-      colnames(table_data[["RHF"]]) == "RHF"] <- "treatment_units"
+  if (!rlang::is_empty(table_data[[config[["data"]][["column"]][["unit_name"]][["rhf"]]]])) {
```

Comment:
Will it not work with `config$data$column$unit_name$rhf`

Reply:
I think both notations work the same way. which one do you prefer?